src/t_zset.c

```diff
@@ -2121,6 +2121,11 @@ void zuiClearIterator(zsetopsrc *op) {
     }
 }
 
+void zuiDiscardDirtyValue(zsetopval *val) {
+    if (val->flags & OPVAL_DIRTY_SDS)
+        sdsfree(val->ele);
```

Comment:
```suggestion
    if (val->flags & OPVAL_DIRTY_SDS) {
        sdsfree(val->ele);
        val->flags &= ~OPVAL_DIRTY_SDS;
        val->ele = NULL;
    }
```
Would this be safer? if one day somebody calling `zuiNewSdsFromValue` or `zuiNext` again might access unknown memory.

Reply:
thanks, safer